person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  # Attribute accessor setters and getters
+  attr_accessor :id, :name, :age
+
+  # Constructor
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # private method
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+  end
+
+  # public method
+
+  def can_use_services?
+    return true if @age || @parent_permission
+  end
+end
```

Comment:
- [ ] In the way you define the `can_use_services?` method, is also defined as a `private` one. I think it would be better if you move it to come before `line 15` so by default it will be a `public` method or add `public` just  before `line 23`.
For example:
```ruby
class Person  
   ...
   # public method

  def can_use_services?
    return true if @age || @parent_permission
  end

  # private method

  private

  def of_age?
    return true if @age >= 18
  end
end